### ISSUE TITLE
Target platform label in status bar does not have 4.5:1 color contrast ratio

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#275]
### Steps to Reproduce

1. Launch Xamarin Workbook application. 
2. Create a new workbook and check the colour contrast ratio of Text (Working Platform) in top centre using colour contrast analyser

### Expected Behavior
Colour contrast ratio should be minimum 4.5:1

### Actual Behavior
Colour contrast ratio is less than 4.5:1 i.e., 3.7:1
as shown in the attached screen-shots.

### User Impact:
Low vision user will feel difficulty in seeing the text.

### Environment

MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9)

### Log File
![issue 4](https://user-images.githubusercontent.com/22956130/39861805-4fd74a1a-5431-11e8-843a-1b802559188a.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1832
